app/services/cocina_support.rb

```diff
@@ -141,10 +141,12 @@ def self.clean(obj)
   def self.update_version_and_lock(cocina_object:, version:, lock:)
     params = Cocina::Models.without_metadata(cocina_object).to_h
     params['version'] = version
-    params[:structural][:contains].each do |file_set_params|
-      file_set_params[:version] = version
-      file_set_params[:structural][:contains].each do |file_params|
-        file_params[:version] = version
+    if params[:structural].present?
```

Comment:
This was necessary as editing a collection fails (at least locally) with an error related to not having structural cocina.